specification/synapse/resource-manager/Microsoft.Synapse/preview/2021-06-01-preview/examples/DeleteWorkspace.json

```diff
@@ -7,6 +7,9 @@
   },
   "responses": {
     "202": {
+      "headers": {
+        "location": true
```

Comment:
location value should be a string

Reply:
Resolved by changing to string

Reply:
please use a reasonable value, not empty value